### DISCUSSION TITLE
How to create KIOSk windows app using windowsAppSdk

### DISCUSSION THREAD

**Discussion by VarunTwri** [microsoft/WindowsAppSDK#2776] [Q&A]
https://github.com/microsoft/WindowsAppSDK/discussions/2692

**Comment by BenJKuhn** [comment]
I asked around a bit, and I couldn't find a definitive answer, but it sounds like the directions here should still be applicable for a packaged application using Windows App SDK.

https://docs.microsoft.com/en-us/windows/configuration/kiosk-single-app

Have you tried that already, and if so, are there specific issues that you're running into?

Ben

**Comment by gus33000** [comment]
There was a backlog item on the product board for support kiosk mode, and apparently it says that only UWP apps can support it.

https://portal.productboard.com/winappsdk/1-windows-app-sdk/c/62-support-single-app-kiosk

Is this information on product board still current/up to date? it looks like product board has not been updated for a while.

**Reply by BenJKuhn** [reply]
Thanks. I assume it's still accurate, but I'm not sure more specifically what the limitation or underlying issue is. Let me ask around a bit more now that I have a breadcrumb to follow.

Ben

**Reply by myokeeh** [reply]
I'd like to know as well. This is currently a blocker for a UWP app I just converted to WinUI 3 / WindowsAppSDK.

**Comment by henningms** [comment]
Any update on this? :D
